documentation/docs/pages/docs/references/rpc-node-providers.mdx

```diff
@@ -33,4 +33,48 @@ You can use [chainlist](https://chainlist.org/) to find all the providers which
 
 ## Local nodes
 
-rindexer should work with any local nodes you run including [Anvil](https://book.getfoundry.sh/anvil/) and [Hardhat](https://hardhat.org/)
\ No newline at end of file
+rindexer should work with any local nodes you run including [Anvil](https://book.getfoundry.sh/anvil/) and [Hardhat](https://hardhat.org/)
+
+# RPC Proxy and Caching
+
+[eRPC](https://erpc.cloud/) is a fault-tolerant EVM RPC proxy and re-org aware permanent caching solution. It is built with read-heavy use-cases in mind such as data indexing and high-load frontend usage.
+
+# Quickstart
```

Comment:
you can also do a shoutout in https://rindexer.xyz/docs/start-building/yaml-config/networks#rpc about this as in turn this is load balancing the request as well which rindexer doesn't have by default so this can be its default